src/reference/asciidoc/aggregator.adoc

```diff
@@ -89,66 +81,54 @@ public AbstractCorrelatingMessageHandler(MessageGroupProcessor processor, Messag
-Hence the `Collection<Message>` variable in the POJO will be cleared too, if passed out of the aggregator.
-If you wish to simply release that collection as-is for further processing,
-it is required that you build a new `Collection` (e.g. `new ArrayList<Message>(messages)`)
-Starting with _version 4.3, the Framework no longer copies the messages to a new collection, to avoid undesired extra
-object creation.
+IMPORTANT: The `SimpleMessageGroup.getMessages()` method returns an `unmodifiableCollection`.
+Therefore, if your aggregating POJO method has a `Collection<Message>` parameter, the argument passed in is exactly that `Collection` instance and, when you use a `SimpleMessageStore` for the aggregator, that original `Collection<Message>` is cleared after releasing the group.
+Consequently, the `Collection<Message>` variable in the POJO is cleared too, if it is passed out of the aggregator.
+If you wish to simply release that collection as-is for further processing, you must build a new `Collection` (for example, `new ArrayList<Message>(messages)`).
+Starting with version 4.3, the framework no longer copies the messages to a new collection, to avoid undesired extra object creation.
 
 
-If the `MessageGroupProcessor` 's `processMessageGroup` method returns a collection, it must be a collection of
-`Message<?>` s.
-In this case, the messages are released individually.
-Prior to _version 4.2_, it was not possible to provide a `MessageGroupProcessor` using XML configuration, only POJO
-methods could be used for aggregation.
-Now, if the framework detects that the referenced (or inner) bean implements `MessageProcessor`, it is used as the
-aggregator's output processor.
+If the `processMessageGroup` method of the `MessageGroupProcessor` returns a collection, it must be a collection of `Message<?>` objects.
+In this case, the messages are individually released.
+Prior to version 4.2, it was not possible to provide a `MessageGroupProcessor` by using XML configuration.
+Only POJO methods could be used for aggregation.
+Now, if the framework detects that the referenced (or inner) bean implements `MessageProcessor`, it is used as the aggregator's output processor.
 
-If you wish to release a collection of objects from a custom `MessageGroupProcessor` as the payload of a message, your
-class should extend `AbstractAggregatingMessageGroupProcessor` and implement `aggregatePayloads()`.
+If you wish to release a collection of objects from a custom `MessageGroupProcessor` as the payload of a message, your class should extend `AbstractAggregatingMessageGroupProcessor` and implement `aggregatePayloads()`.
 
-Also, since _version 4.2_, a `SimpleMessageGroupProcessor` is provided; which simply returns the collection of
-messages from the group, which, as indicated above, causes the released messages to be sent individually.
+Also, since version 4.2, a `SimpleMessageGroupProcessor` is provided.
```

Comment:
Well, @garyrussell selected and insisted on the style to cursive the version whenever it is possible.
What made you to went away from that?
Is that a style applied in other Spring projects?

I don't have objection though, but still... Feels like unnecessary change.

Reply:
I don't recall why we started doing that; I don't mind either way - I certainly don't think it's necessary to change them all back.

Reply:
Yes, I'm fine with that too.
More over I see that SF doesn't use cursive for its version numbers.

Reply:
Code should always be in the code font, which is done with  single back ticks in Asciidoc. Otherwise, quotation marks should be curly quotation marks, which is done with single back ticks inside of double quotations marks in Asciidoc. The only use for double quotation marks is when they appear in the code being quoted.

Reply:
I think this is not the place you have been intended to answer, @Buzzardo, but anyway it doesn't matter: we agreed with Gary to stay with your change.